src/components/Summary/index.js

```diff
@@ -27,7 +27,6 @@ const initialState = {
   privacy: {
     requiresPrivacyConsent: true,
     privacyLabel: '',
```

Comment:
Could maybe refactor this to no longer use the reducer? There is not much state to manage